package.json

```diff
@@ -24,6 +24,7 @@
     "fix": "./scripts/format"
   },
   "dependencies": {
+    "@noble/hashes": "^1.5.0",
```

Comment:
for future reference it'd be very helpful to say why you chose this dependency.

all I see are green flags in their [readme](https://www.npmjs.com/package/@noble/hashes), audited & lots of weekly downloads so 👍